webapp/security/views.py

```diff
@@ -271,9 +280,9 @@ def cve_index():
 
     # Query parameters
     order_by = flask.request.args.get("order-by", default="oldest")
-    query = flask.request.args.get("q", default="")
-    priority = flask.request.args.get("priority", default="")
-    package = flask.request.args.get("package", default="")
+    query = flask.request.args.get("q")
```

Comment:
:+1: thanks for simplifying this.

Reply:
Yeah, After your refactoring, having default values was not relevant anymore